cli/cmd/proxy/envoy/http.go

```diff
@@ -12,11 +14,13 @@ import (
 	"github.com/hashicorp/consul-k8s/cli/common"
 )
 
+var ErrNoLoggersReturned = errors.New("No loggers were returned from Envoy")
+
 // EnvoyConfig represents the configuration retrieved from a config dump at the
 // admin endpoint. It wraps the Envoy ConfigDump struct to give us convenient
 // access to the different sections of the config.
 type EnvoyConfig struct {
-	rawCfg    []byte
+	RawCfg    []byte
```

Comment:
so this change and changing the tests to utilize the `testdata` directory rather than embedding are the only substantial changes here other than the move